queries/subsquid/general/collectionListWithSearchProfile.graphql

```diff
@@ -0,0 +1,36 @@
+#import "../../fragments/collection.graphql"
+#import "../../fragments/collectionDetails.graphql"
+
+query collectionListWithSearch(
+  $first: Int!
+  $offset: Int
+  $search: [CollectionEntityWhereInput!]
+  $orderBy: [CollectionEntityOrderByInput!] = [blockNumber_DESC]
+) {
+  collectionEntities(
+    orderBy: $orderBy
+    limit: $first
+    offset: $offset
+    where: { AND: $search, burned_eq: false }
+  ) {
+    ...collection
+    ...collectionDetails
+    nfts(where: { burned_eq: false }) {
+      id
+      metadata
+      name
+      price
+      burned
+      currentOwner
+      events(where: { interaction_eq: BUY }) {
+        meta
+      }
+    }
+  }
```

Comment:
can the query be slimmed down?
do we need the nfts?
do we need the events?

Reply:
yes, to fix @prury issue